docs/source/release/v6.5.0/index.rst

```diff
@@ -16,7 +16,25 @@ Mantid 6.5.0 Release Notes
 
 We are proud to announce version 6.5.0 of Mantid.
 
-**TODO: Add paragraph summarizing big changes**
+This release includes a wide array of updates, bugfixes, and new features, which build on the functionality of Mantid.
```

Comment:
@perenon @peterfpeterson The list of release highlights is quite ISIS-focussed. Is there anything you'd like included from your ends?

Reply:
> @perenon @peterfpeterson The list of release highlights is quite ISIS-focussed. Is there anything you'd like included from your ends?

We discussed that during our internal meeting, then @gui-co and @darominski had a look at the release notes. For us, everything is OK.
For information, there is a significant amount of pending ILL features (that will certainly go in 6.6), that may explain the relatively low level of ILL specific changes into 6.5.

Cheers,